Indexes/GreenhousePostingPartIndexHandler.cs

```diff
@@ -10,8 +13,19 @@ public override Task BuildIndexAsync(GreenhousePostingPart part, BuildPartIndexC
         {
             context.DocumentIndex.Set($"{nameof(GreenhousePostingPart)}.{nameof(GreenhousePostingPart.Department)}", part.Department, DocumentIndexOptions.Store);
             context.DocumentIndex.Set($"{nameof(GreenhousePostingPart)}.{nameof(GreenhousePostingPart.Location)}", part.Location, DocumentIndexOptions.Store);
-            context.DocumentIndex.Set($"{nameof(GreenhousePostingPart)}.{nameof(GreenhousePostingPart.UpdatedAt)}", part.UpdatedAt, DocumentIndexOptions.Store);
-
+            context.DocumentIndex.Set($"{nameof(GreenhousePostingPart)}.{nameof(GreenhousePostingPart.UpdatedAt)}", part.UpdatedAt, DocumentIndexOptions.Store);
+
+            foreach (var metadataField in part.GetMetadata())
+            {
+                if(metadataField.Value?.Length > 0)
```

Comment:
Use `.Any()` instead of `.Length > 0`. Also need a space after if (`if(` should be `if (`).

Reply:
Sometimes value was null, therefore added a extra check

Reply:
That's fine, so it can be `if (metadataField.Value?.Any())`.

Reply:
this asks me to use a explicit cast, as it cannot convert type bool? to bool.

Reply:
`if (metadataField.Value?.Any() ?? false)`.

Reply:
not sure if you've seen the new update but i used `metadataField.Value?.Any() ?? false` instead